perfkitbenchmarker/linux_benchmarks/terasort_benchmark.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2014 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs a jar using a cluster that supports Apache Spark.
+
+This benchmark takes runs the Apache Terasort benchmark on an
+Apache Spark cluster. The Apache Spark cluster can be one
+supplied by a cloud provider, such as Google's Dataproc.
```

Comment:
This benchmark runs the Apache Hadoop MapReduce Terasort benchmark on an Apache Hadoop YARN cluster. The cluster can be one supplied by a cloud provider, such as Google Cloud Dataproc.

edit as you like